org.eclipse.xtext.ide/src/org/eclipse/xtext/ide/server/concurrent/RequestManager.java

```diff
@@ -51,18 +51,26 @@ public void shutdown() {
 		parallel.shutdown();
 		cancel();
 	}
+	
+	protected void setParallel(ExecutorService parallel) {
```

Comment:
We could also make the fields protected. What do you think?

Reply:
am not sure. if prefer protected setter getter over protced fields